src/pretix/api/serializers/item.py

```diff
@@ -398,6 +425,8 @@ def create(self, validated_data):
             ItemAddOn.objects.create(base_item=item, **addon_data)
         for bundle_data in bundles_data:
             ItemBundle.objects.create(base_item=item, **bundle_data)
+        for program_times_data in program_times_data:
+            ItemProgramTime.objects.create(item=item, **program_times_data)
```

Comment:
It’s not wrong, but IMHO unusual to use the same name for the local var as the looped over var, so I would change it according to the scheme from bundles and addons:

```suggestion
        for program_time_data in program_times_data:
            ItemProgramTime.objects.create(item=item, **program_time_data)
```